client/__tests__/setup.js

```diff
@@ -1,3 +1 @@
-require('./cssModulesFix')
-require('./shimImageLoader')
-require('babel-core/register')
+// require('./specHelper')
```

Comment:
Not in the boilerplate, but more than likely in any project that uses it.  I left this here to show both places where it would need to be added back in.